packages/core/src/components/Box/Box.types.ts

```diff
@@ -0,0 +1,22 @@
+export type Disabled = "opacityDisabled";
```

Comment:
What is this for? Can we avoid it? It doesn't seem to be used

Reply:
Yep, removed it